Scripts/Core/EditorVR.cs

```diff
@@ -96,6 +96,7 @@ static void ResetPreferences()
 
 		void Awake()
 		{
+			ObjectUtils.hideFlags = defaultHideFlags;
```

Comment:
I think you might break existing functionality this way, but haven't tested. By moving this to awake, the default hide flags would not be set in time for the editing context manager on initialization. So, you'll need to test whether things are hidden correctly on first start of the an editorvr installation. My guess is no.

Also, IIRC static constructors should be called even on re-compile, so it's odd that this is not getting set.